templates/lxc-download.in

```diff
@@ -48,17 +46,17 @@ DOWNLOAD_USE_CACHE="false"
 DOWNLOAD_VALIDATE="true"
 DOWNLOAD_VARIANT="default"
 
+DOWNLOAD_KEYID="0xE7FB0CAEC8173D669066514CBAEFF88C22F6E216"
+if [[ -z "$DOWNLOAD_KEYSERVER" ]]; then
+    DOWNLOAD_KEYSERVER="hkp://pool.sks-keyservers.net:80"
```

Comment:
I hadn't realized there were different pools. Looking it up, I notice the explicit group wherein port-80 enabled servers are.

I'm not sure if limiting the pool to p80.sks-keyservers.net is desirable ; if not, I would like to suggest incorporting just a change where we check before overriding the keyserver, as it would be good to be able to set this in .profile etc. I can make a fresh branch and PR for that if preferable.